src/main/com/fulcrologic/rad/report.cljc

```diff
@@ -376,6 +376,27 @@
         (start-report! app report-class {:route-params route-params})
         (comp/transact! app [(dr/target-ready {:target report-ident})])))))
 
+(s/def ::arglist (s/and vector?
+                        (fn [[this props]]
+                          (and (not= this props)
+                               (not= this '_)
+                               (not= props '_)))))
+
+(s/def ::columns (s/coll-of symbol? :kind vector?))
```

Comment:
they will usually use symbols here, but I don't think it should be a hard requirement.

Reply:
Well, I just copied your current 

```clojure
(req! &env sym options ::columns #(every? symbol? %))
```
which I believe also happens at macro-expansion time?!